swift/ITSClient/Tests/ITSMobilityTests/MobilityTests.swift

```diff
@@ -0,0 +1,50 @@
+/*
+ * Software Name : ITSClient
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * see the "LICENSE.txt" file for more details or https://opensource.org/license/MIT/
+ *
+ * Software description: Swift ITS client.
+ */
+
+import Foundation
+import ITSCore
+import Testing
+@testable import ITSMobility
+
+struct MobilityTests {
+    private let mobility: Mobility
+    private let mobilityConfiguration: MobilityConfiguration
+
+    init() throws {
+        mobility = Mobility()
+        let mqttClientConfiguration = MQTTClientConfiguration(host: "test.mosquitto.org",
+                                                              port: 1883,
+                                                              clientIdentifier: UUID().uuidString,
+                                                              useSSL: false)
+        let url = try #require(URL(string: "http://localhost:4318"))
+        let telemetryClientConfiguration = TelemetryClientConfiguration(url: url,
+                                                                        serviceName: "its-tests-service")
+        let coreConfiguration = CoreConfiguration(mqttClientConfiguration: mqttClientConfiguration,
+                                                  telemetryClientConfiguration: telemetryClientConfiguration)
+        mobilityConfiguration = MobilityConfiguration(coreConfiguration: coreConfiguration,
+                                                      stationID: UInt32.random(in: 0..<UInt32.max))
+    }
+
+    @Test("Send position should send a payload on a topic computed from coordinates")
+    func send_position_should_send_payload_on_topic_computed_from_coordinates() async throws {
+        try await mobility.start(mobilityConfiguration: mobilityConfiguration)
+        try await mobility.sendPosition(latitude: 43.63516355648167,
+                                        longitude: 1.3744570239910097,
+                                        altitude: 155,
+                                        heading: 45,
+                                        speed: 8.1)
+        try await mobility.stop()
+        // Wait a bit for the spans flush
+        try await Task.sleep(for: .seconds(0.5))
```

Comment:
It would be so much better if `Mobility.stop()` was blocking:
* there is no guarantee that the SDK has enough time to flush the spans in any "reasonnable" sleep duration;
* it would remove the requirement to do that active wait by sleeping for a=_all_ callers.

Of course, I see this is not related to this commit, indeed, and I missed that in a previous review (all this async stuff is weird to me).

Reply:
The OpenTelemetry swift SDK doesn't wait the completion of the network calls. That's why we must wait a bit. I can do this on our side but I will add an arbitrary value to wait.